packages/utils/tool-utils/src/snapshotNormalizer.ts

```diff
@@ -8,7 +8,7 @@ import {
 	BlobTreeEntry,
 	TreeTreeEntry,
 } from "@fluidframework/driver-utils/internal";
-import { ITree, ITreeEntry, TreeEntry } from "@fluidframework/protocol-definitions";
+import { ITree, ITreeEntry, TreeEntry } from "@fluidframework/driver-definitions/internal";
```

Comment:
I see TreeEntry is an enum. No potential issues from using the same enum but defined in a different package, I imagine? (Like the same class from different packages would not pass `instanceof` checks).

Reply:
Good thought.  TypeScript does not allow instanceof on enum types.  Comparisons between two enums is an error that would require a cast (no casts were added in this PR.)

...but the real answer is that we're working on ensuring the client types and server types never mix.  This PR gets as pretty close.